Localization/en-US/home.yaml

```diff
@@ -2,31 +2,75 @@
 url: "https://foldingathome.org/"
 language: "en-US"
 text: [
+# I AM One IN A MILLION bold text
+"I AM One IN A MILLION",
+
+# Associated text
+"Regardless if you are already folding or haven't heard a word about it before, we need your help to reach our goal -- which is 1 million folders.",
+
+#Associated square button
+"START FOLDING NOW",
+
+#Associated text with the aobve square button
+"Hide this and watch the video.",
+
 # START FOLDING NOW bold text
 "START FOLDING NOW",
 
-# Associated normal text
+# Associated text
 "Find the version of the software you prefer and get started. Downloading Folding@home is completely free, easy to install and safe to use.
-
 Available for:
-
 Linux
-
 Windows
-
 Mac",
 
+#Associated square button
+"START FOLDING",
+
 #JOIN THE COMMUNITY bold text
 "JOIN THE COMMUNITY",
 
 # Associated text
 "The Folding@Home community is spread over countless forums, websites and social media.
-Here’s a list of our official channels.
-For tech support: "
+Here's a list of our official channels.
+For tech support: ",
 
 #URLs - Facebook and Twitter have localised name in certain languages:
 "foldingforum.org",
 "Facebook",
-"Twitter"
+"Twitter",
+
+#LATEST POSTS bold text
+"LATEST POSTS",
+
+# WHY WE NEED YOU bold text
+"WHY WE NEED YOU",
+
+# Associated text
+"Folding@home is a project focused on disease research. The problems we're solving require so many computer calcul¡©ations ? and we need your help to find the cures!",
+
+#FIGHT DESIEASES TOGETHER WITH US bold text
+"FIGHT DISEASES TOGETHER WITH US.",
+
+#Associated text
+"The Folding@home software allows you to share your unused computer power with us ? so that we can research even more potential cures.",
```

Comment:
```suggestion
"The Folding@home software allows you to share your unused computer power with us so that we can research even more potential cures.",
```